kong/db/schema/metaschema.lua

```diff
@@ -206,6 +206,15 @@ local field_schema = {
       { message = { type = "string", required = true } },
       { removal_in_version = { type = "string", required = true } },
       { old_default = { type = "any", required = false } },
+      { replaced_with = { type = "array", required = false,
+          elements = { type = "record",
+            required = false,
+            fields = {
+              { path = { type = "array", len_min = 1, required = true, elements = { type = "string"}} },
```

Comment:
Why `path` is required? I mean that if use just wants to do the `function`?

Reply:
Is there reason we need it to be function?

Reply:
Why would they want just to use `function` ? 🤔

Reply:
Actually what `function` do you mean? `reverse_mapping_function` or `func` ? 🤔